test/testsuite.sh

```diff
@@ -28,15 +28,28 @@ echo "        TOMCAT_CYCLE_COUNT=$TOMCAT_CYCLE_COUNT"
 echo "        ITERATION_COUNT=$ITERATION_COUNT"
 echo "        IMG=$IMG"
 echo "        HTTPD_IMG=$HTTPD_IMG"
+if [ ! -z ${HTTPD_IMG+x} ]; then
+echo "        MPC_CONF=$MPC_CONF"
+fi
 
 if [ ! -d logs ]; then
     mkdir logs
 fi
 
 . includes/common.sh
 
-httpd_create  > /dev/null 2>&1 || exit 2
-tomcat_create > /dev/null 2>&1 || exit 3
+if [ -d target ]; then
+  echo "please run setup-dependencies.sh before!!!"
```

Comment:
`"Missing dependencies. Please run setup-dependencies.sh and then try again"`, WDYT?